services/QuillLMS/app/controllers/teachers/classroom_manager_controller.rb

```diff
@@ -22,9 +23,11 @@ def lesson_planner
 
   def assign
     session[GOOGLE_REDIRECT] = request.env['PATH_INFO']
+    session[CLEVER_REDIRECT] = request.env['PATH_INFO']
```

Comment:
Not a major deal, but any reason to not make this generic between the two providers?, e.g. `session[PROVIDER_REDIRECT]` or whatever.

Reply:
It's a good idea, but I think I'll save it for a future refactor.